app/assets/javascripts/components/plant_population_submission.coffee

```diff
@@ -12,6 +12,14 @@ window.PlantPopulationSubmission = class PlantPopulationSubmission extends Submi
     @$('.population-type').select2(@defaultSelectOptions)
     @$('.plant-line-list').select2(@plantLineListSelectOptions)
 
+    fields = [
+      'establishing-organisation'
+    ]
+
+    $.each fields, (_, field) =>
+      @$(".#{field}").select2(@defaultSelectOptions)
+      @$(".#{field}-wrapper").comboField()
```

Comment:
@kammerer I think I could use some help with this one. The way I see it, the `field_with_errors` class isn't added to the combo field wrapper because it has no `name` attribute, so the code in `errors.coffee` fails:

`attrs = $(form).find('[name]:enabled').not('[type=hidden]').map(-> this.name)`

...however I'm unsure how to add the proper attribute (`comboField()` does not appear to accept it). Could you assist?